modules/ocf/manifests/init.pp

```diff
@@ -5,7 +5,9 @@
   include ocf::etc
   include ocf::firewall
   include ocf::groups
-  include ocf::hidepid
+  if ($::hostname != 'fallingrocks') {
```

Comment:
Could you leave a comment why we are not including this on fallingrocks?

Reply:
I'm running [nginx amplify](https://amplify.nginx.com/login) on fallingrocks for additional insights, since nginx's `stub_status` module is indeed a stub compared to `mod_status` on Apache. The agent runs as the www-data user, and it needs to know the pid of the master nginx process.